src/layouts/UserLayout.vue

```diff
@@ -28,7 +28,7 @@ onUnmounted(() => {
   <div class="d-flex">
     <AppSidebar :sidebarItems="sidebarItems" />
     <div class="d-flex flex-column flex-grow-1 min-vh-100 col">
-      <AppHeader v-if="route.path !== '/app/trends'" />
+      <AppHeader v-if="route.path !== '/app/trends' && route.path !== '/app/profile'" />
```

Comment:
there is no reason to add to much logic in a the template, please create a computed property